### PR TITLE
Fixes #209

### LINKED ISSUE #209: adjlist doesn't create adjlist for each vertices passed in
My first ever use of `Graphs.jl` gives the following error.

```{julia}
g = adjlist([1,2])
add_edge!(g, 1, 2)
# BoundsError
```
I think `adjlist{V}(vs::Vector{V}; is_directed::Bool=true) = AdjacencyList{V}(is_directed, vs, 0, Vector{V}[])` in 
(https://github.com/JuliaLang/Graphs.jl/blob/master/src/adjacency_list.jl#L29) is not correct as `AdjacencyList.AdjList` is not fully initialized with the vertices in `vs`.

Forgive me if I misunderstand the usage, and I will submit a PR if someone can help confirm the issue.

### PR DESCRIPTION
**Pull request by papamarkou**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
-
-doc/build
\ No newline at end of file
+.project
```
Comment (kmsquire) [inline .gitignore]: Generally, .gitignore changes shouldn't be added unless they affect all users.

Comment (scidom) [inline .gitignore]: Thanks @kmsquire, I removed this line.

**Comment by tkelman** [comment]
This is #216 with a few more tests, right? A more descriptive commit message would be good.

**Comment by scidom** [comment]
Yes, that's correct @tkelman; I wanted to expand the test a bit. Is it better to accept the PR you mentioned and then create a new one just for the test?

**Comment by tkelman** [comment]
If the existing PR is acceptable as-is but you'd just like to add or adjust new tests on top of it, then I'd say yes, may as well keep the git authorship of that PR and merge it first. Or you could cherry-pick that commit into this branch and preserve the authorship that way, but the former option is probably easier.

**Comment by scidom** [comment]
Ok, in this case, I will close this PR, will accept #216 and then will add the thorough test via a separate PR.

**Comment by scidom** [comment]
Although it seems that I have neither push access nor I am able to accept PRs, the permissions might have not been set properly or I am doing sth wrong from my end

**Comment by tkelman** [comment]
Hm. You're marked as an organization member but apparently not a member of the group that has commit access here. I added you as a collaborator specifically to this repo, hopefully should work now.

**Comment by scidom** [comment]
Yes, it seems to be ok now @tkelman, I can see the option to accept PRs. I will accept the one we discussed and then will open a new one for the relevant test.

**Pull request closed without merging**